devtools/docker-compose.yml

```diff
@@ -135,6 +135,27 @@ services:
     command:
       server --address :9090 -C /configs/minio /export
 
+  # IMAP worker for remote identities
+  imap_worker:
```

Comment:
it's an imap worker for now, but will evolve to others protocols in future. So imap_worker is maybe not the correct naming

- identity_poller
- identity_worker

are explicit naming